deploy/blobfuse-proxy/blobfuse-proxy.yaml

```diff
@@ -54,8 +54,10 @@ spec:
             systemctl start blobfuse-proxy
             echo "removing the file: /tmp/blobfuse-proxy-v0.1.0.deb"
             rm /tmp/blobfuse-proxy-v0.1.0.deb
-            echo "sleeping for 19 years"
-            sleep 600000000s
+            echo "sleeping for 10 secs"
+            sleep 10s
```

Comment:
that means it can only get logs in the exact time of after 10s, is there better way to get all logs after all tests completed?

Reply:
it is getting all the logs. In the above case, the sleep is to wait for the blobfuse-proxy service to come up. Although it happens much quicker than that.